### ISSUE TITLE
npm test errors with unhandled error event

### ISSUE THREAD

**Issue by tsteenbe** [heremaps/oss-review-toolkit#964]
Running `npm run test` in ./reporter-web-app will result in error below.

```
Determining test suites to run...2018-10-17 10:25 node[82268] (FSEvents.framework) FSEventStreamStart: register_with_server: ERROR: f2d_register_rpc() => (null) (-22)
2018-10-17 10:25 node[82268] (FSEvents.framework) FSEventStreamStart: register_with_server: ERROR: f2d_register_rpc() => (null) (-22)
events.js:167
      throw er; // Unhandled 'error' event
      ^

Error: EMFILE: too many open files, watch
    at FSEvent.FSWatcher._handle.onchange (internal/fs/watchers.js:120:28)
Emitted 'error' event at:
    at FSEvent.FSWatcher._handle.onchange (internal/fs/watchers.js:126:12)
npm ERR! code ELIFECYCLEnpm ERR! errno 1
npm ERR! @here/oss-review-toolkit-reporter@0.0.1 test: `node scripts/test.js --env=jsdom`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the @here/oss-review-toolkit-reporter@0.0.1 test script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/thsteenb/.npm/_logs/2018-10-17T08_25_41_993Z-debug.log
```

**Comment by tsteenbe** [comment]
Solution to this issue is to install [watchman](https://facebook.github.io/watchman/docs/install.html) per https://github.com/facebook/create-react-app/issues/4918

**Comment by tsteenbe** [comment]
Fixes this issue by disabling watchman see [#d5f7004b](https://github.com/heremaps/oss-review-toolkit/commit/d5f7004b48afc8793f880b4afa4183ea16351821) following https://jestjs.io/docs/en/troubleshooting#watchman-issues

**Comment by SuperAL** [comment]
I already had watchman installed, fixed this error by running `brew upgrade watchman`
